src/Symfony/Component/Security/Core/Authorization/Voter/ExpressionVoter.php

```diff
@@ -85,7 +85,7 @@ private function getVariables(TokenInterface $token, $object)
         $variables = array(
             'token' => $token,
             'user' => $token->getUser(),
-            'object' => $object,
+            'object' => $subject,
```

Comment:
Okay, renamed `object` to `subject` here and introduced a new `subject` variable in #16755.